rbac/internal/views.py

```diff
@@ -77,9 +84,57 @@
 )
 from api.utils import RESOURCE_MODEL_MAPPING, get_resources
 
+# environment variables
+client_id = os.getenv("CLIENT_ID")
```

Comment:
Those variable could be obtained also in settings.py and the you can variable in code,  it is fine that it will be here - you can use `ENVIRONMENT.get_value` here - we are doing some for other secret variables.